Traceback (most recent call last):
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\site-packages\jupyter_cache\executors\utils.py", line 51, in single_nb_execution
    executenb(
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\asyncio\base_events.py", line 647, in run_until_complete
    return future.result()
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\chjch\AppData\Local\ESRI\conda\envs\arcpro-py3-clone\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
hw_name = "Changjie's Homework"
num = 1
hw_name + num
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mTypeError[0m                                 Traceback (most recent call last)
[1;32m<ipython-input-21-1ff5e0b5fab7>[0m in [0;36m<module>[1;34m[0m
[0;32m      1[0m [0mhw_name[0m [1;33m=[0m [1;34m"Changjie's Homework"[0m[1;33m[0m[1;33m[0m[0m
[0;32m      2[0m [0mnum[0m [1;33m=[0m [1;36m1[0m[1;33m[0m[1;33m[0m[0m
[1;32m----> 3[1;33m [0mhw_name[0m [1;33m+[0m [0mnum[0m[1;33m[0m[1;33m[0m[0m
[0m
[1;31mTypeError[0m: can only concatenate str (not "int") to str
TypeError: can only concatenate str (not "int") to str

